### PR TITLE
Add uniqueness check for choice enum

### PR DESCRIPTION
**Pull request by rubickcz**

### CODE REVIEW

File: enumfields/enums.py [hunk 1]
```diff
@@ -36,6 +36,9 @@ def __new__(mcs, name, bases, attrs):
                 if k != 'value':
                     setattr(m, k, v)
 
+        # uniqueness check - choices must be unique
+        unique(obj)
```
Comment (rubickcz) [inline enumfields/enums.py]: It does not work for child classes when used the standard way (as decorator), so I am calling it here, in metaclass. The effect is the same and it works for child classes as well.

**Review by jsilhan — APPROVED** [review]

File: enumfields/enums.py (hunk 1 above)
Comment (matllubos) [reply enumfields/enums.py]: good job

Comment (matllubos) [reply enumfields/enums.py]: you can use here 

`return unique(obj)`

**Comment by rubickcz** [comment]
The failing test seems to be unrelated to this PR.

**Pull request merged**